mantidproject/mantid
# Sequential Fitting in Muon FDA interface

**Description of work.**
This PR adds a Sequential Fitting tab to the FDA interface. It is one of the WIMDA replacement requirements.

**To test:**
Open FDA and load `MUSR62260`
Calculate FFT
Do some normal fitting on the Fitting tab
Then try fitting on the Sequential fitting tab

Also make sure sequential fitting still works in the Muon Analysis GUI

Fixes #32036

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
